soc/intel/intel_adsp/common/include/adsp_debug_window.h

```diff
@@ -41,12 +41,16 @@
 #define ADSP_DW_SLOT_SIZE		ADSP_DW_PAGE_SIZE
 #define ADSP_DW_SLOT_COUNT		15
 
+/* debug logs slots usage */
+#define ADSP_DW_SLOT_NUM_CTF		1
```

Comment:
can we call this _TRACE and be format agnostic? we might want to push some different type of trace format, I do not think there is anything CTF specific about this.